functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/pages/appadmin/UserManagerPage.groovy

```diff
@@ -0,0 +1,50 @@
+package org.rundeck.tests.functional.selenium.pages.appadmin
+
+import groovy.transform.CompileStatic
+import org.openqa.selenium.By
+import org.rundeck.tests.functional.selenium.pages.BasePage
+import org.rundeck.util.container.SeleniumContext
+
+/**
+ * User manager page
+ */
+@CompileStatic
+class UserManagerPage extends BasePage {
```

Comment:
this class should go in PRO module since there is no User Manager for OSS

Reply:
updated!